.circleci/config.yml

```diff
@@ -250,7 +250,7 @@ workflows:
           filters:
             branches:
               only:
-                - develop
+                - main
```

Comment:
Seems like this should run on the `master` branch as well, so we could reuse the `main_master_rc_only` alias here. We can do this in a later PR though.